src/utils/time.ts

```diff
@@ -27,15 +38,20 @@ export function addTimedelta(date: Date, components: DateTimeComponents) {
 }
 
 export function subtractTimedelta(date: Date, components: DateTimeComponents) {
-    return addTimedelta(date, negateTimeComponents(components));
+    return addTimedelta(date, negateComponents(components));
 }
 
+/**
+ * @deprecated Use {@link negateComponents} instead
+ */
 export function negateTimeComponents(components: DateTimeComponents) {
-    const negComponents:TimeComponents = {}
+    return negateComponents(components);
+}
+
+export function negateComponents(components: DateTimeComponents) {
```

Comment:
Given that the parameter type is on the parameter, maybe just `negate` for brevity?

Reply:
Sure.